README.md

```diff
@@ -1,12 +1,57 @@
-# go-tracer
 
-## Required Tools:
+![CircleCI](https://circleci.com/gh/lumigo-io/go-tracer-beta/tree/master.svg?style=svg&circle-token=421fefe82bcad1c17c4116f154e25e32ebc90f2c)
+[![Go Report Card](https://goreportcard.com/badge/github.com/lumigo-io/go-tracer-beta)](https://goreportcard.com/report/github.com/lumigo-io/go-tracer-beta)
+[![GoDoc](https://godoc.org/github.com/lumigo-io/go-tracer-beta?status.svg)](https://godoc.org/github.com/lumigo-io/go-tracer-beta)
+
+# go-tracer (BETA)
+
+This is lumigo/go-tracer-beta, Lumigo's Golang agent for distributed tracing and performance monitoring.
+
+## Installation
+
+`go-tracer-beta` can be installed like any other Go library through `go get`:
+
+```console
+$ go get github.com/lumigo-io/go-tracer-beta
+```
+
+Or, if you are already using
+[Go Modules](https://github.com/golang/go/wiki/Modules), you may specify a
+version number as well:
+
+```console
+$ go get github.com/lumigo-io/go-tracer-beta@master
+```
+
+## Usage
+
+You need a lumigo token which you can find under the `Project Settings` and `Tracing` tab in lumigo portal. Then you need just to wrap your Lambda:
+
+```go
+type MyEvent struct {
+  Name string `json:"name"`
+}
+
+func HandleRequest(ctx context.Context, name MyEvent) (string, error) {
+  return fmt.Sprintf("Hello %s!", name.Name ), nil
+}
+
+func main() {
+	wrappedHandler := lumigotracer.WrapHandler(HandleRequest, &lumigotracer.Config{
+		Token:       "<your-token>",
+	})
+	lambda.Start(wrappedHandler)
+}
+```
+
```

Comment:
They also need to add the environment variable: `LUMIGO_USE_TRACER_EXTENSION: true`.
And the layer: `arn:aws:lambda:us-east-1:114300393969:layer:lumigo-tracer-extension:36`.
In a serverless.yml file it looks like this:
```
    environment:
      LUMIGO_USE_TRACER_EXTENSION: true
    layers:
      - arn:aws:lambda:us-east-1:114300393969:layer:lumigo-tracer-extension:36
```

Reply:
btw we should add the same layers table as in python tracer actually. Like this https://github.com/lumigo-io/python_tracer#with-lambda-layers